test/e2e/storage/flexvolume.go

```diff
@@ -25,10 +25,11 @@ import (
 	"time"
 
 	. "github.com/onsi/ginkgo"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
corev1

Reply:
This is just a goimports change; not an attempt to really alias it. Do you really want me to change the default alias here or you were just disagreeing with my the idea that I explicitly chose one (and it was v1)?

By the way, this is the motivation for https://github.com/kubernetes/kubernetes/pull/77407 ; the change has nothing to do with this PR but is inevitably discussed.

Reply:
Standard style is to call this corev1

https://github.com/kubernetes/kubernetes/search?utf8=%E2%9C%93&q=corev1&type=

Reply:
FWIW it looks like that is not quite the case. I'm happy to change it but this was why #77045 got created; `v1` and `corev1` get used roughly equally, but unaliased is about 5x as popular.

My approach on #77407 was to run goimports on a bunch of files to make them consistent but that would have pushed the `v1` alias. If you prefer the corev1 I can do that instead. (It is the goimports that adds the v1 alias, not the editor itself though I had heard that as well but confirmed goimports itself is the perpetrator)

```
$ rg '^[\s]*v1 "k8s.io/api/core/v1"' | wc -l
     220
$ rg '^[\s]*corev1 "k8s.io/api/core/v1"' | wc -l
     257
$ rg '^[\s]*"k8s.io/api/core/v1"' | wc -l
    1130
```

Reply:
@fejta Are you OK if I leave the `v1` alias in _this_ PR and let #77045 handle squaring all those?

The problem is that I am touching 32 different files and if I want to add an alias to these it is going to probably double the size of this PR even though it has nothing to do with ssh.

The explicit `v1` alias didnt require code changes at each of the lines using something from that package but `corev1` is a much bigger change.

Reply:
Sure SGTM